hardhat.config.ts

```diff
@@ -50,8 +50,9 @@ const config: HardhatUserConfig = {
   solidity: {
     compilers: [
       {
-        version: "0.8.17",
+        version: "0.8.20",
         settings: {
+          evmVersion: "paris",
           optimizer: { enabled: true, runs: 800 },
           viaIR: true,
```

Comment:
can now be false

Reply:
do you mean optimizer?

Reply:
no. meant viaIR. but saw other comment somewhere

Reply:
yeah we should still keep it enabled, so that actual deployment still use via ir